packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -557,25 +562,6 @@ contract PerpetualPositionManager is FundingRatePayer, FundingRateApplier {
         return;
     }
 
-    /**
-     * @notice Drains any excess balance of the provided ERC20 token to a pre-selected beneficiary.
-     * @dev This will drain down to the amount of tracked collateral and drain the full balance of any other token.
-     * @param token address of the ERC20 token whose excess balance should be drained.
-     */
-    function trimExcess(IERC20 token) external nonReentrant() returns (FixedPoint.Unsigned memory amount) {
```

Comment:
were you able to remove this because of `gulp`?

Reply:
nvm. the gulp function acts to sync up the `cumulativeFeeMultiplier`, and does not transfer anything. can you explain why you removed this function?

Reply:
I initially removed it because I was relying on periodically calling gulp to deal with refunds. Now that the refunds are gone, it was just removed to save bytes. Will attempt to re-add. I think we should _either_ have gulp or trimExcess. My preference would be gulp as I think it has a smaller impact on bytecode and is more consistent with the emp design.